coach/src/navigation/settings.coffee

```diff
@@ -14,5 +14,6 @@ settings =
     logout: DEFAULT_PATTERN
     default: '{prefix}{base}'
     close: '{prefix}'
+    'second-semester-registration': DEFAULT_PATTERN
 
```

Comment:
medium bug, since we didn't a route defined for 'second-semester-registration', it would error when the link on the banner was clicked